layouts/index.html

```diff
@@ -61,11 +61,8 @@ <h3><a href="{{ .RelPermalink }}">{{ upper .Title }}</a></h3>
         {{ if .Site.Params.fullPostContent }}
           <p>{{ .Content | markdownify }}</p>
 
-
```

Comment:
Not that important, just something to keep prospectively in mind: Running prettier is a `style:` commit type according to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).

Reply:
@alexanderdavide Thanks for pointing out this mistake. I'll keep that in mind when performing future commits.